src/ir/module-utils.h

```diff
@@ -111,10 +111,10 @@ inline void clearModule(Module& wasm) {
 // call this redirect all of its uses.
 template<typename T> inline void renameFunctions(Module& wasm, T& map) {
   // Update the function itself.
-  for (auto& pair : map) {
-    if (Function* F = wasm.getFunctionOrNull(pair.first)) {
-      assert(!wasm.getFunctionOrNull(pair.second) || F->name == pair.second);
-      F->name = pair.second;
+  for (auto& [first, second] : map) {
```

Comment:
It would be good to give these more descriptive names like `func` and `name` now that this is possible.

Reply:
`map` here is `std::map<Name, Name>` usually, So I renamed it as `[name1, name2]`